.changeset/rude-squids-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'graphql-yoga': patch
+---
+
+Improve error messages in case of `operatinName` related errors.
```

Comment:
What was the error message before?
I also don't see any new tests that ensure that the new error message is used and a regression in the future is avoided.